components/motor/i2cmotors/ezopmp.go

```diff
@@ -25,10 +25,9 @@ import (
 
 // AttrConfig is user config inputs for ezopmp.
 type AttrConfig struct {
-	BoardName   string `json:"board"`
-	BusName     string `json:"bus_name"`
-	I2CAddress  *byte  `json:"i2c_address"`
-	MaxReadBits *int   `json:"max_read_bits"`
+	BoardName            string `json:"board"`
+	MaxReadBits          *int   `json:"max_read_bits"`
+	*board.I2CAttrConfig `json:"i2c_attributes"`
```

Comment:
needs user facing config change, but no one external is using this to my knowledge.